table.go

```diff
@@ -201,13 +221,26 @@ var TableList = []Table{
 		Model:               &miner.MinerLockedFund{},
 		NetworkVersionRange: AllNetWorkVersions,
 	},
+
+	// up to actors v8/nv16 only, ctx: https://github.com/filecoin-project/lily/issues/1076
+
 	{
 		Name:                "miner_pre_commit_infos",
 		Schema:              1,
 		Task:                tasktype.MinerPreCommitInfo,
 		Model:               &miner.MinerPreCommitInfo{},
-		NetworkVersionRange: AllNetWorkVersions,
+		NetworkVersionRange: ToNetworkVersion16,
+	},
+
+	// added from nv17
+
+	{
+		Name:                "miner_pre_commit_infos_v9",
+		Schema:              1,
+		Task:                tasktype.MinerPreCommitInfo,
+		NetworkVersionRange: FromNetworkVersion17,
 	},
```

Comment:
@frrist can I get a sanity check here? I know there's a new schema in Lily but the task name stays the same. Should I still split it like so or is it fine to *only* use the new `miner_pre_commit_infos` task based on this [issue](https://github.com/filecoin-project/lily/issues/1076) and assume it'll also work pre-nv17?

Reply:
`miner_pre_commit_infos_v9` isn't a valid task name. The `miner_pre_commit_infos` task will populate the `miner_pre_commit_infos` table for all networks prior to nv17, and will populate the `miner_pre_commit_infos_v9` table for nv17 and beyond.